File: nx-gradle/spring-boot/g-sb-app/src/main/java/com/example/gsbapp/HelloController.java
package com.example.gsbapp;

import com.example.gsblib.HelloService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class HelloController {

  @Autowired
  private HelloService helloService;

  @GetMapping("/")
  public String greeting() {
    return helloService.message();
  }
}

File: nx-maven/spring-boot/m-sb-kt-app/src/main/kotlin/com/example/msbktapp/HelloController.kt
package com.example.msbktapp

import com.example.msbktlib.HelloService
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class HelloController(
  @Autowired val helloService: HelloService,
) {
  @GetMapping("/")
  fun greeting(): String = helloService.message()
}
